frontend/ui/widget/opdsbrowser.lua

```diff
@@ -58,7 +58,37 @@ local OPDSBrowser = Menu:extend{
 }
 
 function OPDSBrowser:init()
-    self.item_table = self:genItemTableFromRoot(self.opds_servers)
+    local servers = G_reader_settings:readSetting("opds_servers") or {}
+    if next(servers) == nil then -- If there are no saved servers, add some defaults
+        servers = {
+          {
+	     title = "Project Gutenberg",
+	     url =
+	    "http://m.gutenberg.org/ebooks.opds/?format=opds",
+          },
+          {
+	     title = "Feedbooks",
+	     url =
+	    "http://www.feedbooks.com/publicdomain/catalog.atom",
+          },
+          {
+    	     title = "ManyBooks",
+    	     url = "http://manybooks.net/opds/index.php",
+          },
+          {
+    	     title = "Internet Archive",
+    	     url =
+    	    "http://bookserver.archive.org/catalog/",
+          },
+        }
+	for i in ipairs(servers) do
+            fields = {servers[i].title, servers[i].url}
+            self:addServerFromInput(fields) 
```

Comment:
The way I set it up is that if opds_servers was empty, it assumes it is initializing a default list of catalogs. This should only happen in two cases I can imagine: Upon the initial loading of the browser and upon the deletion of all feeds. This should save them to disk so that they are not reinitalized and it lets the user delete whichever of the defaults they want.